Receiver/BatteryStreamReceiver/TemperatureStatastics.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BatteryStreamReceiver
+{
+    public class TemperatureStatastics
```

Comment:
This class has mainly setters and getters. It indicates a missed opportunity for encapsulation. What about including the statistics computation/collection also in this class? That way, addition of new parameters will get localized.